sdk_contrib/async_processor/src/main/java/io/opentelemetry/sdk/contrib/trace/export/DisruptorEventQueue.java

```diff
@@ -51,6 +52,20 @@ public void translateTo(
               event.setEntry(arg0, arg1);
             }
           };
+  private static final EventTranslatorThreeArg<
+          DisruptorEvent, ReadableSpan, EventType, CountDownLatch>
+      TRANSLATOR_THREE_ARG =
+          new EventTranslatorThreeArg<DisruptorEvent, ReadableSpan, EventType, CountDownLatch>() {
+            @Override
+            public void translateTo(
+                DisruptorEvent event,
+                long sequence,
+                ReadableSpan arg0,
+                EventType arg1,
+                CountDownLatch arg2) {
+              event.setEntry(arg0, arg1, arg2);
```

Comment:
I would change the order EventType, ReadableSpan, CDL

Reply:
I kept the same order of the TWO_ARG event. Should we change also that one? 
I also prefer having EventType as first argument

Reply:
So then let's change to have EventType as the first argument then. I would change all calls :)